examples/users.sql

```diff
@@ -0,0 +1,60 @@
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+DROP TABLE IF EXISTS `grouplist`;
+CREATE TABLE `grouplist` (
+  `username` varchar(50) NOT NULL,
+  `gid` int(11) NOT NULL DEFAULT 0
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+DROP TABLE IF EXISTS `groups`;
+CREATE TABLE `groups` (
+  `gid` int(11) NOT NULL,
+  `name` varchar(30) NOT NULL DEFAULT '',
+  `status` char(1) DEFAULT 'A',
+  `password` varchar(64) NOT NULL DEFAULT 'x'
+) ENGINE=InnoDB DEFAULT CHARSET=latin1 ROW_FORMAT=COMPACT;
+
+DROP TABLE IF EXISTS `users`;
+CREATE TABLE `users` (
+  `username` varchar(50) NOT NULL DEFAULT '',
+  `name` varchar(32) NOT NULL DEFAULT '',
+  `shell` varchar(20) NOT NULL DEFAULT '/bin/bash',
+  `password` varchar(160) DEFAULT '',
+  `status` char(1) NOT NULL DEFAULT 'N',
+  `uid` int(11) NOT NULL,
+  `gid` int(11) NOT NULL DEFAULT 100,
+  `gecos` varchar(128) NOT NULL DEFAULT '',
+  `homedir` varchar(32) NOT NULL DEFAULT '/bin/sh',
```

Comment:
Thanks for this. It looks good on the whole. I'd just prefer the defaults in the table to be more secure, just in case someone does a copy/paste and doesn't read it through. Perhaps the homedir should have no default.